requirements.txt

```diff
@@ -1,3 +1,4 @@
--r pytorch-requirements.txt
 -r build-requirements.txt
+-r pytorch-nightly-requirements.txt
+-r torchvision-nightly-requirements.txt
 -r test-requirements.txt
```

Comment:
Nit: What do you think about (a separate PR for) moving all of the *-requirements.txt files to a directory?  I confess I added most of those files, but I am beginning to think that we have too many in the top-level directory.